src/modules/displayTask.js

```diff
@@ -0,0 +1,20 @@
+function displayTask() {
+  const todoList = document.querySelector('.todo-list');
+  const localData = JSON.parse(localStorage.getItem('tasks'));
+  if (localData !== null) {
+    localData.forEach((task) => {
+      todoList.innerHTML += `
+              <div class="edit-task" data-key="${task.index}"> 
+              <ul class="ul-edit" >
+                  <li class="tick" ><input id="${task.index}" type="checkbox"></li>
+                  <li class"text-value" >${task.text}</li>
+                  <li class="remove-btn" id="${task.index}"><i class="fa-solid fa-ellipsis-vertical fa-lg vertical-dot"></i></li>
+              </ul>  
+              <hr>
+              </div>
+              `;
+    });
+  }
+}
+
+export default displayTask;
```

Comment:
- [ ] Kindly ensure your function dynamically renders that hardcoded array of tasks.

Reply:
@TedLivist  Thank you for the feedback. can you help me with an idea to do that?

Reply:
@principles31 from your implementation I can see that you are fetching the localStorage and iterating over its data. Here, I would suggest that instead of fetching tasks inside this function, you should pass your list of tasks as an argument instead and iterate over that task.

```
  function displayTask(tasksArray) {
    const todoList = document.querySelector('.todo-list')
    if (tasksArray !== null) {
      tasksArray.forEach((task) => {
        todoList.innerHTML += `
          ...
        `
      })
    }
  }
```
OR
```
  function displayTask(tasksArray, todoList) {
    if (tasksArray !== null) {
      tasksArray.forEach((task) => {
        todoList.innerHTML += `
          ...
        `
      })
    }
  }
```

Reply:
This way you can pass in the array of tasks or localStorage array (in a future project)

Reply:
@TedLivist  I appreciate,i will implement that to see how it works